plugins/woocommerce/client/legacy/css/woocommerce.scss

```
@@ -1838,6 +1838,7 @@ p.demo_store,
 	&::before {
 		font-family: "WooCommerce";
 		content: "\e028";
+		content: "\e028" / ""; // Add an empty alternative text so screen readers ignore it
```

> I update the CSS on this commit to tix it: [Add empty alt text for pesudo-element content](https://github.com/woocommerce/woocommerce/pull/50132/commits/3177a9ee7d008f0fd1bf883927389338969a2694)

Great finding @Manussakis!

Is it intended to duplicate the property? Shouldn't we have just one?

```suggestion
		content: "\e028" / ""; // Add an empty alternative text so screen readers ignore it
```

---

Thanks, @kmanijak !
Yes, we do need it duplicated in case there's an assistive technology that doesn't support alternative text for pseudo-elements. In that case, the first option will be used, and the `content` property will still have a valid value.

---

Ah, right! Thanks for the explanation!